CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+The development tracker for odbc_fdw is on GitHub:
+http://github.com/cartodb/odbc_fdw
+
+---
+
+1. [General](#general)
+2. [Bugs and Features](#bugs-and-features)
+3. [Submitting contributions](#submitting-contributions)
+
+---
+
+# General
+
+Every new feature (as well as bugfixes) should come through a [github pull requests](https://help.github.com/articles/using-pull-requests) and should have test cases for it.
+
+Go [here](https://github.com/CartoDB/odbc_fdw/tree/master/test#connectors-integration-tests) to learn how the test framework works and how to add new tests to the project.
+
```

Comment:
Slightly out of scope: I'd add a small section about submitting good bug reports. Also a template comes in handy.